h/views/admin/groups.py

```diff
@@ -47,23 +47,31 @@ def groups_index(context, request):
     permission="admin_groups",
 )
 class GroupCreateView(object):
+    """View handlers for creating new groups"""
+
     def __init__(self, request):
-        user_svc = request.find_service(name="user")
-        list_org_svc = request.find_service(name="list_organizations")
-        self.organizations = {o.pubid: o for o in list_org_svc.organizations()}
+        self.request = request
+
+        self.user_svc = self.request.find_service(name="user")
+        self.list_org_svc = self.request.find_service(name="list_organizations")
+        self.group_create_svc = self.request.find_service(name="group_create")
+        self.group_members_svc = self.request.find_service(name="group_members")
```

Comment:
Yep, I think this is a nicer way of doing this than having each method retrieve them itself (and takes advantage of the fact that these methods are grouped into a class).